src/main/kotlin/com/exactpro/th2/conn/dirty/tcp/core/api/IChannel.kt

```diff
@@ -117,25 +117,33 @@ interface IChannel {
         val acceptAllCerts: Boolean = false,
     )
 
-    enum class SendMode(val handle: Boolean, val mangle: Boolean) {
+    enum class SendMode(val handle: Boolean, val mangle: Boolean, val socketSend: Boolean, val mstoreSend: Boolean) {
```

Comment:
```suggestion
    enum class SendMode(val handle: Boolean, val mangle: Boolean, val socketSend: Boolean, val mqPublish: Boolean) {
```

Reply:
@Nikita-Smirnov-Exactpro I think your option is better. Should we apply it?